cli/flags.rs

```diff
@@ -494,6 +505,16 @@ pub fn parse_flags(
     flags = parse_run_args(flags.clone(), run_matches);
   }
 
+  // TODO (matt): Do we also need to set a v8 flag here too? - Seems not required
+  // TODO (matt): handle the inspector host/port flags here
```

Comment:
Port is most important if you'd like to debug more then one `deno` process at the same time, it should be possible to set different ports for them.

Reply:
Thanks @ak239 this is now possible via `--debug=127.0.0.1:5555`